Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -94,33 +126,53 @@ private void ServerAlert(EntityUid uid, ServerAlertsComponent status, OnTemperat
                     break;
 
                 // Heat mild.
-                case <= 345 and > 335:
+                case <= 360 and > 335:
                     status.ShowAlert(AlertType.Hot, 2);
                     break;
 
                 // Heat strong.
-                case > 345:
+                case > 360:
                     status.ShowAlert(AlertType.Hot, 3);
                     break;
             }
         }
 
-        private void ChangeDamage(EntityUid uid, TemperatureComponent temperature, OnTemperatureChangeEvent args)
+        private void EnqueueDamage(EntityUid uid, TemperatureComponent component, OnTemperatureChangeEvent args)
+        {
+            ShouldUpdateDamage.Add(component);
+        }
+
+        private void ChangeDamage(EntityUid uid, TemperatureComponent temperature)
         {
             if (!EntityManager.TryGetComponent<DamageableComponent>(uid, out var damage))
                 return;
 
-            if (args.CurrentTemperature >= temperature.HeatDamageThreshold)
+            // See this link for where the scaling func comes from:
+            // https://www.desmos.com/calculator/0vknqtdvq9
+            // Based on a logistic curve, which caps out at MaxDamage
+            var coldK = 0.01;
+            var heatK = 0.005;
+            var a = 1;
+            var y = temperature.DamageCap;
+            var c = y * 2;
```

Comment:
Given that it can't go below absolute zero, the `ColdDamageThreshold` of 260 already caps the cold damage at a damage value less than `MaxDamage`. I guess this isn't really a problem, but might still be confusing. So either mention that explicitly or make the cold damage use a different function that reaches the max when the `diff == ColdDamageThreshold`? 

Also I guess these numbers should either just be `const`s or data-fields?

Reply:
Also minor balance nitpick. Currently cold damage only starts dealing damage at `-13c` and only reaches 1 damage per second once it reaches ~ `-40c`.

Given the players are usually just wearing everyday work clothes, I feel like the damage should start earlier and get to 1dps faster?

Reply:
yeah, i'll tweak it a bit